Updated Storage for KIS

Goldenpsp opened the pull request:
Set MKS module base to 5000-  I came to this number by taking the standard KIS container, which holds 1000 and stacked 4 next to an MKS base module.  The MKS base module is a bit larger than for KIS containers so 5000 seemed fair.
Set Mk3 to 6500- MK3's are slightly larger so I gave them slightly more space.
changed inflatable Ag and Hab to KISItem and storage size to 2500 so 2
can be fit in a MKS base.  These two inflatables were KAS attachable before and can be KIS attached without modification.  Unfortunately letting KIS calculate the volume automatically caused them to be far too large to fit in any storage container.  I seem to recall being able to fit 2 into an MKS base before, so i set their KIS storage size to 2500 so you can fit 2 in an MKS base.

enceos commented:
I second this pull request. Current value of 50L is even smaller than Kerbal's pocket.

pectus-solentis commented:
I third this pull request, and I have one more thing i hope to say...

It seems to be, THESE VOLUMES FOR INFLATABLES EVEN BE CALCULATED AS IF THESE ARE ALREADY DEPLOYED.

Goldenpsp commented:
Pectus.  Is all caps really necessary?  It is quite rude.  As I already explained in the above comment I changed the KIS storage volume for the inflatable HAB and agriculture modules with a KIS override config rather than let KIS calculate manually, as it does calculate on mesh size.  I am actually playing with the expandotubes but will probably do those as a separate pull request.  They do fit in the new resized containers but they are a bit oversized compared to before.

Goldenpsp commented:
Rover

I did some more tweaking, namely to the Inflatables.  I adjusted settings on the inflatable HAB and Ag slightly.  I also added ModuleKISItem to the expandotubes, flexotubes, and docking port.  This was to adjust their stored size to be more appropriate to their KAS counterparts.
